DataFormats/METReco/interface/HcalPhase1FlagLabels.h

```diff
@@ -0,0 +1,32 @@
+#ifndef DataFormats_METReco_HcalPhase1FlagLabels_h
+#define DataFormats_METReco_HcalPhase1FlagLabels_h
+
+#include "DataFormats/METReco/interface/HcalCaloFlagLabels.h"
+
+namespace HcalPhase1FlagLabels
+{
+    enum HBHEStatusFlag {
+        /*  0 */ HBHEHpdHitMultiplicity = HcalCaloFlagLabels::HBHEHpdHitMultiplicity,
+        /* 11 */ HBHEIsolatedNoise = HcalCaloFlagLabels::HBHEIsolatedNoise,
+        /* 12 */ HBHEFlatNoise = HcalCaloFlagLabels::HBHEFlatNoise,
+        /* 13 */ HBHESpikeNoise = HcalCaloFlagLabels::HBHESpikeNoise,
+        /* 15 */ HBHETS4TS5Noise = HcalCaloFlagLabels::HBHETS4TS5Noise,
```

Comment:
It is good to maintain proper history of flag settings. We intend
to change some bit settings, not just add the new ones.

On 08/04/2016 05:17 PM, Slava Krutelyov wrote:
>
> In DataFormats/METReco/interface/HcalPhase1FlagLabels.h <https://github.com/cms-sw/cmssw/pull/15324#discussion_r73612580>:
>
> > @@ -0,0 +1,32 @@
> > +#ifndef DataFormats_METReco_HcalPhase1FlagLabels_h
> > +#define DataFormats_METReco_HcalPhase1FlagLabels_h
> > +
> > +#include "DataFormats/METReco/interface/HcalCaloFlagLabels.h"
> > +
> > +namespace HcalPhase1FlagLabels
> > +{
> > +    enum HBHEStatusFlag {
> > +        /*  0 */ HBHEHpdHitMultiplicity = HcalCaloFlagLabels::HBHEHpdHitMultiplicity,
> > +        /* 11 */ HBHEIsolatedNoise = HcalCaloFlagLabels::HBHEIsolatedNoise,
> > +        /* 12 */ HBHEFlatNoise = HcalCaloFlagLabels::HBHEFlatNoise,
> > +        /* 13 */ HBHESpikeNoise = HcalCaloFlagLabels::HBHESpikeNoise,
> > +        /* 15 */ HBHETS4TS5Noise = HcalCaloFlagLabels::HBHETS4TS5Noise,
>
> why not simply edit HcalCaloFlagLabels?
> partial reuse of the same values looks odd.
>
> IMO, These flags are too high level to pass downstream support of different meanings for phase1 vs not phase1.
> We will need to support re-reco of "legacy" HF.
> It seems simpler to preserve common phase space for flag enums.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/cms-sw/cmssw/pull/15324/files/f2734c7b29b226480f1efc887de88c8ae73f121f#r73612580>, or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AFS4-XtJAMxoTjJTi7ZGZ5hjIRcAfyywks5qcmTegaJpZM4JXio9>.
>